### PR TITLE
fixed disappearing of description when rotate

### PR DESCRIPTION
**Pull request by TenNga**
### PR Checklist:

Hey there! It's great to have you here. Here's a quick checklist to help you make a shiny PR:

* [x] Have you checked out the guidelines in our [Contributing](https://github.com/FightPandemics/FightPandemics/blob/master/CONTRIBUTING.md) document?
* [x] Have you looked if there might be other open [Pull Requests](https://github.com/FightPandemics/FightPandemics/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) for the same update/change?

### Quick Description of the PR

Fix disappearing of description when screen rotate (mobile) issue https://github.com/FightPandemics/FightPandemics/issues/343

### **Issue**:
image

### **Fixed :**
image

### An Overview of the Changes

* Changed media query and also margin to match mockup design

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -77,13 +79,13 @@ const StyledStrapline = styled(StyledWelcome)`
   line-height: 7rem;
```
[the remark this answers is not in the document]
Comment (TenNga) [reply client/src/pages/Home.js:84]: I'm using the largest heading size, we need to add an extra size in theme

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -51,15 +51,14 @@ const MainContainer = styled.div`
...
-const StyledWelcome = styled.h2`
-  font-size: ${theme.typography.heading.three};
+const StyledWelcome = styled.h1`
+
   text-align: left;
   margin: 0 0 7rem 0;
   font-family: ${theme.typography.font.family.display}, sans-serif;
   font-style: normal;
 
   @media only screen and ${mq.phone.wide.max} {
```
Comment (anniegiang) [inline client/src/pages/Home.js:61]: For the mobile screen let's keep the font styles for the title and p tags like the designs:

current
Screen Shot 2020-05-14 at 7 02 24 PM

figma
Screen Shot 2020-05-14 at 7 04 37 PM

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -15,12 +15,13 @@ const FlexChild = styled.div`
   margin-bottom: 2rem;
 `;
 
-const StyledIntro = styled.div`
+const StyledIntro = styled.div` 
+
   @media screen and ${mq.phone.narrow.max} {
```
Comment (joshmorel) [inline client/src/pages/Home.js:20]: Hi @TenNga I'll add my review as I'm trying to help with getting PR's merged. I'd echo @anniegiang comments on consistent styling. To add the margin between the description and links should be consistent(ish) w phone.narrow.
```suggestion
  @media screen and ${mq.phone.wide.max} {
```
If you think `4rem` is too much as it pushes the first button too much down the screen then at least `2rem`. But I don't think that's an issue keeping them the same is probably best.

File: client/src/pages/Home.js (hunk 2 above)
Comment (TenNga) [reply client/src/pages/Home.js:61]: DONE.

File: client/src/pages/Home.js (hunk 3 above)
Comment (TenNga) [reply client/src/pages/Home.js:20]: I used phone wide cause I need to show a mobile version when the screen is under 767px. and the phone narrow doesn't have option for 767px.

Comment (TenNga) [reply client/src/pages/Home.js]: Thank you for your review @joshmorel. I totally agree with you and now I will change narrow to wide right away

**Comment by joshmorel** [comment]
lgtm.

Related but potentially separate issue the `phone.wide.max` of 767px is actually too small for larger phone (Pixel XL, iPhone X*), but increasing that would mean we're overlapping with `tablet.narrow.min` of `768px` which could lead to some issues for tablet. 

For MVP and given we are developing native apps, I don't think we need to worry about tablets much. However, if we do want to match all phones to Mobile design and all Tablets to desktop design regardless of rotation, we'd need to include `orientation` in the media query style rules.

**Pull request merged**